include/Spectra/LinAlg/Orthogonalization.h

```diff
@@ -104,6 +128,17 @@ void JensWehner_orthogonalisation(Matrix& in_output, Eigen::Index leftColsToSkip
     in_output.rightCols(rightColToOrtho) = right_cols;
```

Comment:
Is the final line necessary? I have not tested it, but `right_cols` seems to be a reference to the right cols of `in_output`, which will be modified by `QR_orthogonalisation()` in place.